packages/ra-core/src/controller/ListController.js

```diff
@@ -194,8 +207,18 @@ export class ListController extends Component {
         }
     };
 
+    hideActiveFilters = () => {
+        this.setState({ activeFilters: {} });
+    };
+
+    showInactiveFilters = () => {
```

Comment:
Should this be `showActiveFilters`?

Reply:
@tomknig Maybe you can clarify later what happens here. If we need to change the wording, let's do it in a follow up PR.